tests/fullstack-test/mpp/runtime_filter.test

```diff
@@ -62,5 +74,15 @@ mysql> set @@tidb_isolation_read_engines='tiflash'; set tidb_enforce_mpp = 1; se
 |          1 |
 +------------+
 
+# test timestamp column type for issue #8222
+mysql> set @@tidb_isolation_read_engines='tiflash'; set tidb_enforce_mpp = 1; set tidb_runtime_filter_mode="LOCAL"; select * from test.t1_timestamp, test.t2_timestamp where t1_timestamp.k2=t2_timestamp.k2;
++------+---------------------+------+---------------------+
+| k1   | k2                  | k1   | k2                  |
++------+---------------------+------+---------------------+
+|    1 | 2023-10-20 00:00:00 |    1 | 2023-10-20 00:00:00 |
++------+---------------------+------+---------------------+
```

Comment:
add `alter table xxx compact tiflash replica` to make sure data are flush into stable layer.